protos/orchestrator_service.proto

```diff
@@ -256,6 +257,7 @@ message ScheduleTaskAction {
     string name = 1;
     google.protobuf.StringValue version = 2;
     google.protobuf.StringValue input = 3;
+    map<string, string> tags = 4;
```

Comment:
I was expecting that we'd add `tags` to `TaskScheduledEvent` but not to `TaskCompletedEvent`. What's the usecase for `TaskCompletedEvent`?